packages/cipherstash-proxy-integration/src/extended_protocol_error_messages.rs

```diff
@@ -66,7 +66,11 @@ mod tests {
         if let Err(err) = result {
             let msg = err.to_string();
 
-            assert_eq!(msg, "db error: ERROR: Column 'encrypted_unconfigured' in table 'unconfigured' has no Encrypt configuration. For help visit https://github.com/cipherstash/proxy/blob/main/docs/errors.md#encrypt-unknown-column");
+            // This is similar to below. The error message comes from tokio-postgres when Proxy
+            // returns cs_encrypted_v1 and the client cannot convert to a string.
+            // If mapping errors are enabled (enable_mapping_errors or CS_DEVELOPMENT__ENABLE_MAPPING_ERRORS),
+            // then Proxy will return an error that says "Column X in table Y has no Encrypt configuration"
+            assert_eq!(msg, "error serializing parameter 1: cannot convert between the Rust type `&str` and the Postgres type `cs_encrypted_v1`");
```

Comment:
Unfortunately, this is not as clear now, but with mapping errors disabled (default, and how it is here) we emit a warning, and return the value to the client in `cs_encrypted_v1`, and this is the client (tokio-postgres) reporting an error. Not sure what else we can do to improve

Reply:
I've run into this a couple of times with tokio-postgres and it can be a bit confusing. 
Not much we can do, because is actually tokio enforcing type correctness - the statement types have not been rewritten.